### DISCUSSION TITLE
Seamless animation

### DISCUSSION THREAD

**Discussion by 19Qingfeng** [remix-run/remix#8263] [Q&A]
I have tried many ways, hoping to find a way to achieve seamless animation in remix.

I want to achieve seamless scrolling animation of page switching, When you leave from page An and enter page B. Page A leaves while page B enters.

I'm not sure how to realize the animation of two pages at the same time. I tried to use React-Transiton-Group or Frame-Motion could be realized. I hope I can get some help.🙏🙏🙏

**Comment by rasyidf** [comment]
perhaps, you can use unstable_viewTransition [here](https://remix.run/docs/en/main/components/nav-link#unstable_viewtransition)
